modules/cstrike/csx/meta_api.cpp

```diff
@@ -81,18 +81,18 @@ struct sUserMsg
 	funEventCall func;
 	bool endmsg;
 } g_user_msg[] = {
-	{"CurWeapon",	&gmsgCurWeapon,		Client_CurWeapon,	false},
-	{"Damage",		&gmsgDamage,		Client_Damage,		false},	
-	{"Damage",		&gmsgDamageEnd,		Client_Damage_End,	true},
-	{"WeaponList",	&gmsgWeaponList,	Client_WeaponList,	false},
-	{"ResetHUD",	&gmsgResetHUD,		Client_ResetHUD,	true},
-	{"AmmoX",		&gmsgAmmoX,			Client_AmmoX,		false},
-	{"ScoreInfo",	&gmsgScoreInfo,		Client_ScoreInfo,	false},
-	{"AmmoPickup",	&gmsgAmmoPickup,	Client_AmmoPickup,	false},
-	{"SendAudio",	&gmsgSendAudio,		Client_SendAudio,	false},
-	{"TextMsg",		&gmsgTextMsg,		Client_TextMsg,		false},
-	{"BarTime",		&gmsgBarTime,		Client_BarTime,		false},
-	{"DeathMsg",	&gmsgDeathMsg,		Client_DeathMsg,	false},
+	{"CurWeapon",  &gmsgCurWeapon,  Client_CurWeapon,  false},
+	{"Damage",     &gmsgDamage,     Client_Damage,     false}, 
+	{"Damage",     &gmsgDamageEnd,  Client_Damage_End, true},
+	{"WeaponList", &gmsgWeaponList, Client_WeaponList, false},
+	{"HLTV",       &gmsgHLTV,       Client_HLTV,       false},
+	{"AmmoX",      &gmsgAmmoX,      Client_AmmoX,      false},
+	{"ScoreInfo",  &gmsgScoreInfo,  Client_ScoreInfo,  false},
+	{"AmmoPickup", &gmsgAmmoPickup, Client_AmmoPickup, false},
+	{"SendAudio",  &gmsgSendAudio,  Client_SendAudio,  false},
+	{"TextMsg",    &gmsgTextMsg,    Client_TextMsg,    false},
+	{"BarTime",    &gmsgBarTime,    Client_BarTime,    false},
+	{"DeathMsg",   &gmsgDeathMsg,   Client_DeathMsg,   false},
```

Comment:
Without judging the rest of this patch, you should update your editor settings so stuff like this doesn't happen. These are unnecessary changes.